src/zjs_spi.c

```diff
@@ -215,7 +224,7 @@ static ZJS_DECL_FUNC(zjs_spi_open)
 
     // Default values
     u32_t bus = 0;
-    double speed = 10;
+    double speed = 2;
```

Comment:
this is wrong as on the frdm_k64f, the value will be interpreted as baud rate, and a baud rate of 2 will not work.  In fact, the clock speed should be used here, see #1449